### PR TITLE
fix: Reduce the file size of artifacts that are saved to disk (ENG-3692)

### PR DESCRIPTION
**Pull request by mukundbayyaram**
## Conventional Commit
fix: Reduce the file size of each artifact that will be saved to disk so that the compression happens a bit more faster (as part of [ENG-3692](https://cloudelements.atlassian.net/browse/ENG-3692))

#### File size of Netsuite element that was being saved to disk:
Before: 46 MB
After: 16 MB
This would help make the compression faster as the file read becomes a bit faster

### CODE REVIEW

**Review by GiriNellore — APPROVED** [review]

File: src/utils/saveToFile.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const {logDebug} = require('./logger');
 
 module.exports = curry(async (fileName, data) => {
   try {
-    fsExtra.outputFileSync(fileName, JSON.stringify(await data, null, 2), 'utf8');
+    fsExtra.outputFileSync(fileName, JSON.stringify(await data), 'utf8');
```
Comment (GiriNellore) [inline src/utils/saveToFile.js:8]: **question:** Why should we await on data here?

**Review by GiriNellore — COMMENTED** [review]
@mukundbayyaram Left one comment

**Review by GiriNellore — COMMENTED** [review]
@mukundbayyaram Left one comment

**Review by GiriNellore — CHANGES_REQUESTED** [review]
@mukundbayyaram Not sure why it is showing as approved though I select the comment option. Anyways, trying one more time

**Review by GiriNellore — APPROVED** [review]

File: src/utils/saveToFile.js (hunk 1 above)
Comment (mukundbayyaram) [reply src/utils/saveToFile.js:8]: Addressed by removing the redundant awaits

**Pull request merged**